src/components/button/button.tsx

```diff
@@ -73,7 +73,6 @@ export class Button {
             <button
                 class={{
                     'mdc-button': true,
-                    'mdc-button--unelevated': this.primary,
```

Comment:
I renamed this from `fix` to `refactor`. It could have also been a `chore` I guess. But it's not a bug fix 😄

Reply:
sure thing 👍